node/pkg/governor/governor.go

```diff
@@ -774,7 +774,7 @@ func (gov *ChainGovernor) CheckPendingForTime(now time.Time) ([]*common.MessageP
 						ce.transfers = append(ce.transfers, transfer)
 
 						// Add inverse transfer to destination chain entry if this asset can cancel flows.
-						key := tokenKey{chain: dbTransfer.EmitterChain, addr: dbTransfer.EmitterAddress}
+						key := tokenKey{chain: pe.token.token.chain, addr: pe.token.token.addr}
```

Comment:
Is there a test case we could/should add here to check flow cancelling when loading from pending entries?

Reply:
Yeah that's probably a good idea. In `governor_test.go` there's a test `TestPendingTransferBeingReleased`. I can modify this to also include a flow cancel transfer to make sure that it's included in this flow.

Reply:
@djb15 Added a new unit test in the latest commit

Reply:
Nice! Even cooler that you do it without advancing time and just using the flow cancelling to build the headroom for the queued transfer